### ISSUE TITLE
bubble up logs from commands

### ISSUE THREAD

**Issue by dolanor** [dagger/dagger#2485]
Just to add to this, there does not seem to be a way to "bubble up" logs from when commands are executed inside of a docker container. This is really important for diagnosing why a plan is going wrong. 

```
package simple

import (
    "dagger.io/dagger"
    "universe.dagger.io/alpine"
    "universe.dagger.io/docker"
)

dagger.#Plan & {

    actions: {
        image: alpine.#Build & {
            packages: {}
        }
        run_simple: docker.#Run & {
            input: image.output
	        command: {
				name: "sh"
				flags: "-c": "date" // the date will not show up in your logs!
			}
        }
    }
}
```

_Originally posted by @RonanQuigley in https://github.com/dagger/dagger/issues/2139#issuecomment-1096574018_

**Comment by dolanor** [comment]
@RonanQuigley: I extracted that from the previous issue.
You can display the date in the log when it is in the `plain` log format.
If the task succeeds in the `auto` (or `tty`) log format, it gets folded and hide its log.
Also, if the task doesn't need rerunning thanks to cache, it won't display the log again as well (even in `plain` log format)

Is there something you would like to work differently? (outside of the fact that the default `tty` output is buggy and is currently getting fixed)

**Comment by RonanQuigley** [comment]
Sorry, I've not looked at Dagger for a while now (originally it was for work but that's been put down for the time being). 

If I remember correctly, what I noticed after I posted this was that the logs _were_ there, it's just that they were in grey text which poorly contrasted with the terminal so I found that they were _very_ easy to miss. 

It sounds like you're already aware of the formatting/presentation issues within the terminal, and I presume this is being tracked. If so, feel free to close.
